doc/asciidoc/connected-components.adoc

```diff
@@ -243,32 +247,30 @@ include::scripts/connected-components.cypher[tag=cypher-loading]
 
 `algo.unionFind`
 
-- if a threshold configuration parameter is supplied only relationships with a property value higher then the threshold
-are merged
+* If a threshold configuration parameter is supplied, only relationships with a property value higher than the threshold
+are merged.
 
 `algo.unionFind.queue`
```

Comment:
@mneedham Please could you check that the use of codeblocks in the *Implementations* section is correct